README.md

```diff
@@ -0,0 +1,158 @@
+
+**Contact management**
+- [ ] Invite new contact
+- [ ] Accept a contact request
+
+**1 to 1 messages interaction**
+- [ ] Send 1 to 1 conversation
+- [ ] Subscribe to 1 to 1 conversation
+- [ ] Unsubscribe from a 1 to 1 conversation
+- [ ] Documented API for 1 to 1 conversation
+- [ ] Working examples for 1 to 1 conversations
+
+**Private groups interaction**
+- [ ] Publish messages on private groups
+- [ ] Subscribe to private groups
+- [ ] Unsubscribe from a private groups
+- [ ] Documented API for private groups interaction
+- [ ] Working examples for private groups interaction
+
+## Code Example
+You'll find some code examples on the [examples folder](/examples)
+
+## Installation
+status-go-sdk is a lightweight dependency package, that means by now it does not have any external dependency. You can install it by running:
+```
+go get github.com/status-im/status-go
+```
+
+## API Reference
+TBD
+
+## Tests
+TBD
+
+## How to use?
+`status-go-sdk` relies on a running instance of `statusd`, we can configure it to be accessed on many different ways, but the easiest one is like:
+
+```
+# Clone status-go
+mkdir $GOPATH/src/status-im/
+cd $GOPATH/src/status-im/
+git clone git@github.com:status-im/status-go.git
+cd status-go
+
+# Build it and run it
+make statusgo && ./build/bin/statusd -shh=true -standalone=false -http=true -status
+```
+
+With this you'll have a statusd server running on `8545` port.
+
+`status-go-sdk` receives as parameter an [RPCClient interface](https://github.com/status-im/status-go-sdk/blob/master/sdk.go#L7), this allows you to use it in many different ways, here we will use a basic example relying on go-ethereum/rpc client.
+
+```
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/status-im/status-go-sdk"
+)
+
+func main() {
+  // Here we create a new remote client against our running statusd
+	rpcClient, err := rpc.Dial("http://localhost:8545")
+	checkErr(err)
+	remoteClient := &remoteClient{rpcClient}
+
+  // Setting up a new sdk client
+	client := sdk.New(remoteClient)
+
+  // We signup and login as a new account
+	a, err := client.SignupAndLogin("password")
+	checkErr(err)
+
+  // Joining a new channel
+	ch, err := a.JoinPublicChannel("my_channel")
+	checkErr(err)
+
+  // We will be sending messages to that channel every 3 seconds
+	for range time.Tick(3 * time.Second) {
+		message := fmt.Sprintf("PING : %d", time.Now().Unix())
+		_ = ch.Publish(message)
+	}
+}
+
+// This is the remote client we will use to
+type remoteClient struct {
+	c *rpc.Client
+}
+
+func (rc *remoteClient) Call(req *sdk.Request, res interface{}) error {
+	return rc.c.Call(res, req.Method, req.Params)
+}
+
+func checkErr(err error) {
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+```
+
+If you want to subscribe to this same channel you could easily do it with `Subscribe` method like:
+```
+ch, err := a.JoinPublicChannel("supu")
+checkErr(err)
+
+_, _ = ch.Subscribe(func(m *sdk.Msg) {
```

Comment:
I was able to test out the subscribe method by only using lines 138-146 from here. This is just a suggestion, but maybe its easier to only include lines 138-146 here and say that if they're introduced to the above code example before publishing the 'PING' message, you will be able to subscribe and see the PING-PONG back and forth.

Now that I think of it, it might also be worth it to put this example under `status-go-sdk/examples/readme/main.go` in order to show what this example would look like when completed with the subscribe statement. Although the mitsuku example is pretty similar too, with a few added dependencies.